src/coreclr/gc/gc.cpp

```diff
@@ -295,9 +295,9 @@ size_t gc_heap::bgc_tuning::gen1_index_last_bgc_start;
 size_t gc_heap::bgc_tuning::gen1_index_last_bgc_sweep;
 size_t gc_heap::bgc_tuning::actual_num_gen1s_to_trigger;
 
-gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[2];
-gc_heap::bgc_tuning::tuning_stats gc_heap::bgc_tuning::gen_stats[2];
-gc_heap::bgc_tuning::bgc_size_data gc_heap::bgc_tuning::current_bgc_end_data[2];
+gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[] = {};
```

Comment:
feels like explicit array sizes are better for readability, no?

Reply:
They are redundant in this case. I mentioned that above and I can put them back if that is preferred.

Reply:
I also think using explicit sizes makes it more readable :)

just making sure - 
if I only do 
`gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[2];`
means `memset` will be called on memory `gen_calc[2]` occupies (or some other form of zeroing will happen to make this memory zeroed)

`gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[2] = {};`
means it also calls constructors?

Reply:
> I also think using explicit sizes makes it more readable :)

Okay, will put back.

> if I only do
gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[2];
means memset will be called on memory gen_calc[2] occupies (or some other form of zeroing will happen to make this memory zeroed)

> if I do
gc_heap::bgc_tuning::tuning_calculation gc_heap::bgc_tuning::gen_calc[2] = {};
means it also calls constructors?

The C++ `{}` is defined as "uniform initialization" - see [3](https://en.cppreference.com/w/cpp/language/initialization). It exists because it should always do the "right thing". Because these fields are declared at global scope they will be initialized properly, both will call the default constructor. This means the `= {};` isn't needed, it simply conveys they are being initialized and could be copied as is in a local context and mean the same thing. I can remove them because they aren't needed, but they convey the intent which I generally prefer.